tests/functional/container/test_container.py

```diff
@@ -699,28 +699,43 @@ def test_spare_with_one_broken(self):
         obj_meta = self.json_loads(resp.data)
         # Get a list of chunks for future spare request
         chunks = obj_meta['chunks']
-        if len(self.conf['services']['rawx']) < len(chunks) + 1:
+        if len(chunks) < count_broken + 1:
+            self.skip(
```

Comment:
Minor: `self.skip` is deprecated, we should use `self.skipTest`.
and return is useless as `skip` or `skipTest` raise a `skipException`.